x/auth/client/cli/query.go

```diff
@@ -198,8 +198,7 @@ $ %s query txs --%s 'message.sender=cosmos1...&message.action=withdraw_delegator
 		},
 	}
 
-	cmd.Flags().StringP(flags.FlagNode, "n", "tcp://localhost:26657", "Node to connect to")
-	cmd.Flags().String(flags.FlagKeyringBackend, flags.DefaultKeyringBackend, "Select keyring's backend (os|file|kwallet|pass|test)")
```

Comment:
Technically speaking this is a client-breaking change, so should wait RC before getting merged.

However, I agree with @chengwenxi that this flag is non-sensical here, and I'm pretty sure no-one uses it. So I'm actually okay to merge this PR. Any strong opinions @robert-zaremba and @ryanchristo ?

Reply:
I think it would be safe to merge. Not sure how one would use it in this situation anyways.